presto-docs/src/main/sphinx/develop/presto-native.rst

```diff
@@ -126,4 +126,34 @@ There is also an additional parameter:
     If the time period is less than or equal to the parameter value, the request
     is valid.
     If the time period exceeds the parameter value, the request is rejected as
-    authentication failure (HTTP 401).
\ No newline at end of file
+    authentication failure (HTTP 401).
+
+Async Data Cache and Prefetching
+--------------------------------
+
+``num-connector-io-threads``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type** ``integer``
+    * **Default value:** ``30``
+
+    Size of IO executor for connectors to do preload/prefetch.  Prefetch is
+    disabled if ``num-connector-io-threads`` is set to zero.
+
+``async-data-cache-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type** ``bool``
+    * **Default value:** ``true``
+
+    Whether async data cache is enabled.  Setting ``async-data-cache-enabled``
+    to ``false`` disables split prefetching in table scan.
+
+``async-cache-ssd-gb``
```

Comment:
consistency: async-data-cache-enabled vs. async-cache-ssd-gb - one says data-cache, the other just cache. Would be nice to fix.

Reply:
That will be a big change, rather not changing it here

Reply:
Understood. Perhaps, file a GitHub issue about this.

Reply:
#21185